### PR TITLE
fix(build): use alpine version of node docker image

### PR DESCRIPTION
**Pull request by Defferrard**

### CODE REVIEW

File: .dockerignore [hunk 1]
```diff
@@ -3,8 +3,11 @@ coverage
 dist
 docs
 node_modules
-test
+**/test/**
```
Comment (spaenleh) [inline .dockerignore:6]: Does this also include `fixtures` folders ?

File: docker/Dockerfile [hunk 2]
```diff
@@ -1,49 +1,56 @@
...
+        build-base
+
+COPY ./package.json ./
+
+# Install the application
+RUN yarn install --production --ignore-scripts
+COPY --from=builder ./app/dist ./dist
+
+USER node
+CMD yarn start
```
Comment (spaenleh) [inline docker/Dockerfile:56]: Is it better practice to do `yarn start` than to use `node` on the built server.js ?

**Review by spaenleh — CHANGES_REQUESTED** [review]
Nice work ! 🥡 🐳 

I have a small question about the command used to run the app.

File: docker/Dockerfile (hunk 2 above)
Comment (spaenleh) [reply docker/Dockerfile:57]: Ah I see that `yarn start` is executing `yarn serve` which does exactly `NODE_PATH=./dist node dist/server.js`

File: .dockerignore (hunk 1 above)
Comment (Defferrard) [reply .dockerignore:6]: Yes it does ! Should we keep theses folders in production ? It doesn't seems to be used in the application to me

Comment (spaenleh) [reply .dockerignore:6]: Okay then it is all good ! No they should not be used in production.

**Review by spaenleh — APPROVED** [review]

**Comment by Defferrard** [comment]
`bcrypt` use `node-pre-gyp` to build the dependency. So we need to omit "--ignore-scripts" and add a condition for our `postinstall` script. Otherwise we can prebuild `bcrypt` somewhere else, but it means that we need to do the same for every packages that needs to be prebuild.

**Pull request merged**